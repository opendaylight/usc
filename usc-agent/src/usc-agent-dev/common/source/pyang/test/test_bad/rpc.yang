module rpc1 {
	namespace "urn:test:rpc";
	prefix "rpc";

	rpc activate-software-image {
		input {
		}
		output {
		}
	}

}                                                                                                    



