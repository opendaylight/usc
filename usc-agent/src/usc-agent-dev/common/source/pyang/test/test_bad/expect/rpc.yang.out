rpc.yang:1: warning: WBAD_MODULE_NAME
rpc.yang:6: error: EXPECTED_DATA_DEF
rpc.yang:8: error: EXPECTED_DATA_DEF
